msgio/msgio.go

```diff
@@ -1,108 +0,0 @@
-package main
```

Comment:
Why is this file being deleted?

Reply:
Less code to maintain. But I'll restore it, there's really no motivation to remove it.